CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
+* Modified `skbio.stats.composition.ilr` and `skbio.stats.composition.ilr_inv` to suppress errors when inputing a 1D basis.  ([#1393](https://github.com/biocore/scikit-bio/issues/1393))
```

Comment:
Thanks for expanding, this looks great! Just to confirm, what is the rationale for changing `ilr_inv` handling of `basis` from:

```python
_check_orthogonality(basis)
```

to:

```python
_check_orthogonality(clr_inv(basis))
```
It seems unrelated to handling 1-D basis. Is this a separate bug? If so please add that to the changelog.